modules/apps/accessibility/accessibility-settings-state-web/src/main/resources/META-INF/resources/index.ts

```diff
@@ -29,3 +29,5 @@ type AccessibilityMenuSetting = {
 export const accessibilityMenuAtom = State.atom<
 	Record<KEYS, AccessibilityMenuSetting>
 >('accessibility-menu', {} as any);
+
+export {isReducedMotion} from './isReducedMotion';
```

Comment:
Moving `isReducedMotion` to be able to place `deepClone` in `frontend-js-web`. Without the move, there would be a circular dependency between `frontend-js-web` and `frontend-js-state-web`, as `frontend-js-state-web` needs to use `deepClone`.